### ISSUE TITLE
Negative dimension size caused by subtracting 2 from 1 for '{{node max_pooling2d_1/MaxPool}} = MaxPool[T=DT_FLOAT, data_format="NHWC", explicit_paddings=[], ksize=[1, 2, 2, 1], padding="VALID", strides=[1, 2, 2, 1]](Placeholder)' with input shapes: [?,1,128,128].

### ISSUE THREAD

**Issue by Areaybiu** [AstarLight/Satellite-Segmentation#81]
Negative dimension size caused by subtracting 2 from 1 for '{{node max_pooling2d_1/MaxPool}} = MaxPool[T=DT_FLOAT, data_format="NHWC", explicit_paddings=[], ksize=[1, 2, 2, 1], padding="VALID", strides=[1, 2, 2, 1]](Placeholder)' with input shapes: [?,1,128,128].
这个该怎么解决

**Comment by Areaybiu** [comment]
解决了，能直接跑起来，不能一下子跑起来的都是版本问题，tensorflow-gpu==2.0.0，Keras==2.3.1
